Archivos.c

```diff
@@ -0,0 +1,63 @@
+#include"Archivos.h"
+
```

Comment:
Agregar `iostream`, se utiliza para mostrar por pantalla si el archivo no se pudo abrir
```suggestion
#include <iostream>
```